Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,20 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
-
+//there are 6 varibale declarations
 // b) How many function calls are there?
-
+//There is one function call, console.log
 // c) Using documentation, explain what the expression movieLength % 60 represents
+// according to the documentation the The remainder (%) operator returns the remainder left over when one operand is divided by a second operand. It always takes the sign of the dividend.
```

Comment:
So what does `movieLength % 60` try to calculate?

Reply:
I've updated those comments with ones from GPT that give clearer explanations